psi4/src/psi4/detci/import_vector.cc

```diff
@@ -82,15 +82,15 @@ void CIWavefunction::parse_import_vector(SlaterDetSet *sdset, int *ialplist, int
 
     /* now figure out how the frozen stuff is going to map */
     if (CalcInfo_->num_drc_orbs > alphastrings->ndrc || CalcInfo_->num_drc_orbs > betastrings->ndrc) {
-        outfile->Printf(
-            "(parse_import_vector): Can't drop more orbitals now"
-            " than in the imported guess!\n");
-        abort();
+        const std::string msg = "(parse_import_vector): Can't drop more orbitals now than in the imported guess!\n";
+        outfile->Printf(msg.c_str());
```

Comment:
Not a big critique here, but isn't the `Printf` call here redundant, since throwing should print the message anyway?

Reply:
Yes it _should_. Unfortunately, on some systems that does not happen reliably due to unclear reasons, possibly a `libstdc++` bug or something of that nature. So Psi4 is left with the choice of either printing the message twice or have some users getting no message at all.

I have some ideas on how this might be at least somewhat cleaned up later, but for now I just want to get rid of `abort()` calls.

Reply:
Hmm, I see, I see. I can accept that, then!